hudi-common/src/main/java/org/apache/hudi/common/schema/HoodieSchema.java

```diff
@@ -361,19 +361,6 @@ public static HoodieSchema removeMetadataFields(HoodieSchema schema) {
     return HoodieSchema.fromAvroSchema(resultAvro);
   }
 
-  /**
-   * Creates a nullable union (null + specified type).
-   *
-   * @param type the non-null type
-   * @return new HoodieSchema representing a nullable union
-   */
-  public static HoodieSchema createNullableSchema(HoodieSchema type) {
```

Comment:
There is another method `createNullable` that covers similar functionality so I am deduping